### PR TITLE
Fix versions

### PR DESCRIPTION
**Pull request by dgrisonnet**
Follow up PR from:
https://github.com/brancz/prometheus-example-app/pull/3#issuecomment-595759630
https://github.com/brancz/prometheus-example-app/pull/4#discussion_r388921836

### CODE REVIEW

**Comment by dgrisonnet** [comment]
@lilic @brancz what do you think about this?

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,7 @@
 VERSION=v0.2.0
```
Comment (lilic) [inline Makefile]: Lets add a `VERSION` file to read this variable from that file, I am fine to also do it in a follow up.

**Review by lilic — APPROVED** [review]
Otherwise looks good to me.

File: Makefile (hunk 1 above)
Comment (dgrisonnet) [reply Makefile]: I addressed your comment, what do you think about it? @lilic

**Review by lilic — APPROVED** [review]
lgtm

**Pull request merged**